prepare-serviceworker-list.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const path = require('path');
+
+const outputPath = './src/service-worker-list.json';
+
+// According to the need, add or remove directories from this array
+const targetFolders = [
+  { name: 'core', path: './public/lib/core/' },
```

Comment:
Looks like it doesn't work if I try to content editing. Not sure if we need to explicitly specify `contentEdit`, `officeEditor` and `assets` folders?